test/Lsp.Tests/ClientCapabilityProviderTests.cs

```diff
@@ -18,24 +18,9 @@ namespace Lsp.Tests
 {
     public class ClientCapabilityProviderTests
     {
-        private static readonly Type[] Capabilities = {
-            typeof(CompletionCapability),
-            typeof(HoverCapability),
-            typeof(SignatureHelpCapability),
-            typeof(ReferencesCapability),
-            typeof(DocumentHighlightCapability),
-            typeof(DocumentSymbolCapability),
-            typeof(DocumentFormattingCapability),
-            typeof(DocumentRangeFormattingCapability),
-            typeof(DocumentOnTypeFormattingCapability),
-            typeof(DefinitionCapability),
-            typeof(CodeActionCapability),
-            typeof(CodeLensCapability),
-            typeof(DocumentLinkCapability),
-            typeof(RenameCapability),
-            typeof(WorkspaceSymbolCapability),
-            typeof(ExecuteCommandCapability),
-        };
+        private static readonly Type[] Capabilities = typeof(ClientCapabilities).Assembly.GetTypes()
```

Comment:
I thought it was better to load these via reflection, as the tricky ones where missing in the static list (e.g. SynchronizationCapability).